public/locales/en/translation.json

```
@@ -173,6 +173,30 @@
         }
       }
     },
+    "monitoring": {
+      "title": "Monitoring",
+      "logs": {
+        "title": "Logs",
+        "followLogs": "Follow logs",
+        "logsCount": "received {{count}} log entries",
+        "logsCount_one": "received {{count}} log entry",
+        "tooltips": {
+          "copy": "Copy logs"
+        }
+      },
+      "instances": {
+        "openWorkflowTooltip": "click to open workflow {{name}}",
+        "updateTooltip": "refetch instances",
+        "successfullExecutions": {
+          "title": "Successful Executions",
+          "empty": "There are no recently successful executions."
```

```suggestion
          "empty": "There are no recent successful executions."
```
This wasn't strictly wrong, but a bit confusing. (recently successful = the success happened recently; recent successful executions = successful executions that occurred recently)

---

Good catch. I was just copying from the old design...